srcpkgs/wiremix/patches/useless-git.patch

```diff
@@ -0,0 +1,15 @@
+--- a/build.rs
```

Comment:
the comment I put at the top was important, it gave context as to why we patch it out